README.md

```diff
@@ -857,6 +857,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [Pulsetic](https://pulsetic.com) - 10 monitors, 6 Months of historical Uptime/Logs, unlimited status pages, and custom domains included! For infinite time and unlimited email alerts for free. You don't need a credit card.
   * [Wachete](https://www.wachete.com) - monitor five pages, checks every 24 hours.
   * [Xitoring.com](https://xitoring.com/) — Uptime monitoring: 20 free, Linux and Windows Server monitoring: 5 free, Status page: 1 free - Mobile app, multiple notification channel, and much more!
+  * [OneUptime](https://oneuptime.com/) - Unlimited Status Pages, Unlimited Subscribers, Unlimited Viewers, Unlimited Manual Monitors with alerts, custom branding & custom domain name.
```

Comment:
This mentions that `Active Monitors at $1/month` does it mean without paying this $1 you get no monitoring in fact?